migrations/20170302155852_user.js

```diff
@@ -0,0 +1,20 @@
+exports.up = function(knex, Promise) {
+  return Promise.all([
+    knex.schema.createTable('github_user', function(table) {
+      // The GitHub ID would do as a primary key, but this approach works
+      // better with Bookshelf's `Model.isNew` method.
+      table.increments();
+      table.integer('github_id').notNullable();
+      table.text('name');
+      table.text('login').notNullable();
+      table.text('avatar_url');
```

Comment:
Do we need `avatar_url` stored?
We use it in UI, but we get it from GitHub through session, so not sure if we need to persist it.

On the other hand, as main reason for this table now is login metrics, would it be worth adding some `last_login_date` so we would be able to see which accounts are active?
But this of course may evolve to some bigger "activities" table later...